OpenRA.Mods.Common/Widgets/Logic/Ingame/ProductionTooltipLogic.cs

```diff
@@ -115,7 +115,7 @@ static string ActorName(Ruleset rules, string a)
 		{
 			ActorInfo ai;
 			if (rules.Actors.TryGetValue(a.ToLowerInvariant(), out ai) && ai.HasTraitInfo<TooltipInfo>())
-				return ai.TraitInfo<TooltipInfo>().Name;
+				return ai.TraitInfos<TooltipInfo>().First(Exts.IsTraitEnabled).Name;
```

Comment:
It looks like this method is expected to return `a` if there isn't a valid tooltip, so better to use `FirstOrDefault` and only return if non-null here.